### DISCUSSION TITLE
Character Telemetry

### DISCUSSION THREAD

**Discussion by plusflabby** [EveronLife/EveronLife#200] [Ideas]
**I propose tracking of characters**, and to use this data to display examples like: K/D percents, playing time, and distance traveled. May have a menu like Escape from Tarkov that'll show overall statistics.

Here's a few examples of displayed data:
- average time spent alive
- average kills each session
- average session time

**Comment by thebonbon** [comment]
Good Idea, but since this is not a Gamemode focused on K/D, I think we should not show anything related to Kills / Time alive to the player. Server Owners / Admins should see this data per Player and overall

Maybe more RP related infos to show the player like:
- Money earned (total / per hour)
- Resources gathered
- Distance traveled

**Comment by plusflabby** [comment]
@thebonbon I agree. Some more RP info to shop may be like:

- Time spent in prison
- Favorite resource(s)
- Distance driven
